Planner:
  visualize_potential: false
  allow_unknown: true
  planner_window_x: 0.0
  planner_window_y: 0.0
  default_tolerance: 0.0
  dq: 2.5
  dq_step: 1.25
  run_time: 2
  max_iterations: 200000
  goal_radius: 1
  